### ISSUE TITLE
WAPE calculation

### ISSUE THREAD

**Issue by SuhanG17** [pedrolarben/TimeSeriesForecasting-DeepLearning#2]
Dear author, I just finished reading "An Experimental Review on Deep Learning Architectures for Time Series Forecasting" and I want to reproduce some experiments in my own code with pytorch. However, I can't seem to get my metric "WAPE" to work properly, that is my "WAPE" was suspiciously low. From your code, I understand that you denormalize output from the model, as well as the labels back to raw data first. Then, you calculate "WAPE" for each outputs-and-targets vector pair (`for y, o in zip(actual, predicted):`) and store it in a list (`res[]`). Lastly, we average over the list for the final result, which is the WAPE per sample. Is this result the one reported in Table 11 of your Review? Thanks in advance.

**Comment by pedrolarben** [comment]
Hi Suhan, note that for the sake of readability, the results in table 11 are in percentage %. Hence you will have to multiply by 100 the results obtained by the wape function.

**Comment by SuhanG17** [comment]
I see, the percentage makes much more sense here. I couldn't wrap my head around how error exceeds one by such a large margin. Thank you so much!

**Comment by SuhanG17** [comment]
I have a question on the "N.Layers" for TCN in your appendix file. I'm wondering if the "N.Layers" were meant to denote the number of dense layers or the number of stacks in the TCN network?

**Comment by pedrolarben** [comment]
`N.Layers` refers to the number of TCN residual blocks. It is the `nb_stacks` argument from the [keras-tcn API](https://github.com/philipperemy/keras-tcn#arguments).
